### ISSUE TITLE
Add user-toggleable "hit count heat map" display in source editors

### ISSUE THREAD

**Issue by markerikson** [RecordReplay/devtools#6856]
**Description:**

We fetch all line hit counts for a given file on load. Currently, there's two ways to see that info:

- Hover over a given line and wait for the blue "N hits" hover button
- Look at the "Outline" view and see how many times a given file ran

When I'm skimming a file and really trying to get a sense of how much of the code ran, I really want to see hit counts for _every_ line displayed right there in the editor itself.

In other words, what I'd like is a toggleable version of the per-line display seen in Istanbul's HTML code coverage reports:

![image](https://user-images.githubusercontent.com/1128784/169406083-7370f901-8244-4b31-adb3-d42f3c15ea18.png)

I imagine having another checkbox at the bottom of the source editor or something that would let us toggle this on and off.

**Comment by jcmorrow** [comment]
This seems like something that is not getting done until at least next sprint, pushed to 32

**Comment by gideonred** [comment]
This ticket has been moved to [FE-191](https://linear.app/replay/issue/FE-191/add-user-toggleable-hit-count-heat-map-display-in-source-editors)
